source/ada/lsp-ada_handlers-named_parameters_commands.adb

```diff
@@ -107,21 +108,28 @@ package body LSP.Ada_Handlers.Named_Parameters_Commands is
          Edit.span := (Loc.span.first, Loc.span.first);
          Edit.newText := Name & " => ";
 
-         if Client_Supports_documentChanges then
+         if Message_Handler.Versioned_Documents then
             Edits.documentChanges (1).Text_Document_Edit.edits.Append (Edit);
          else
-            Edits.changes (Edits.changes.First).Append
-              (LSP.Messages.TextEdit (Edit));
+            if Edits.changes.Contains (Self.Where.textDocument.uri) then
+               Edits.changes (Self.Where.textDocument.uri).Append
```

Comment:
I think this will never be called because we already appended an empty vector at line:
```
Edits.changes.Insert (Document.URI, Empty);
```